### ISSUE TITLE
Error on adding new redirect

### ISSUE THREAD

**Issue by donatiss** [the-control-group/voyager-redirects#2]
I have an error when saving a new redirect uri

> SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'qwer' for key 'voyager_redirects_from_unique' (SQL: insert into `voyager_redirects` (`from`, `to`, `type`, `updated_at`, `created_at`) values (qwer, qwerty, 301, 2017-10-05 21:11:40, 2017-10-05 21:11:40))

it's 'cause **from** is UNIQUE but if i remove UNIQUE, then i have saved a few records.

**Comment by donatiss** [comment]
any ideas?

**Comment by chadidi** [comment]
@tsulavi 'from' column should be a unique key. so you can't add duplicate entries.

**Comment by donatiss** [comment]
yeah, I understand that. but when i put inputs from and to, select redirect type and press save, i have this error. i think its a bug.

**Comment by chadidi** [comment]
@tsulavi I have never used this package I search for this package migrations but I didn't find them so I think you have added migration manually so check that your migrations are ok.
